linera-service/src/node_service.rs

```diff
@@ -459,16 +460,16 @@ where
     S: Store + Clone + Send + Sync + 'static,
     ViewError: From<S::ContextError>,
 {
+    #[graphql(cache_control(no_cache))]
     async fn chain(&self, chain_id: ChainId) -> Result<ChainStateExtendedView<S::Context>, Error> {
-        let client = self.clients.try_client_lock(&chain_id).await?;
-        let view = client.chain_state_view(Some(chain_id)).await?;
-        Ok(ChainStateExtendedView::new(view))
+        let view = self.storage.load_chain(chain_id).await?;
```

Comment:
So for the records, we tried to conceal `storage_client` from `linera-service` to make future security audits easier.

Reply:
Then I'd rather expose the `LocalNode` and use that, instead of the `ChainClient`s?